### PR TITLE
fixed the abstract and introduction to make it more related to TVR

### PR DESCRIPTION
**Pull request by zhangli-abcd**

### CODE REVIEW

File: draft-wqb-tvr-applicability.md [hunk 1]
```diff
@@ -69,9 +69,10 @@ informative:
 
 --- abstract
 
-This document describes the applicability of various network management tools
-and data models that may be used for scheduling, to meet the requirements of a set
-of representative use cases described in the "TVR (Time-Variant Routing) Requirements" (I-D.ietf-tvr-requirements).
+This document provides the applicability statement on how the TVR(Time-Variant Routing)
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:72]:
```suggestion
This document provides the applicability statement on how the TVR (Time-Variant Routing)
```

File: draft-wqb-tvr-applicability.md [hunk 2]
```diff
@@ -69,9 +69,10 @@ informative:
...
+data model may be used for scheduling in specific time variant network cases, to meet
+the requirements of a set of representative use cases described in the
+"TVR Requirements" (I-D.ietf-tvr-requirements).
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:75]:
```suggestion
"TVR (Time-Variant Routing) Requirements" (I-D.ietf-tvr-requirements).
```
I think it should be a faithful copy of the title of https://datatracker.ietf.org/doc/draft-ietf-tvr-requirements/.

File: draft-wqb-tvr-applicability.md [hunk 3]
```diff
@@ -90,12 +91,13 @@ of resources to satisfy service demands or adhere to local networks policies.
 Many common building blocks are required for all these cases for adequate
 management of schedules and related scheduled actions.
 
-This document describes the applicability of various network management tools
-and data models that may be used for scheduling, to meet the requirements of a set
-of representative Time-Variant Routing (TVR) use cases described in {{?I-D.ietf-tvr-use-cases}}. By leveraging
-a reference framework presented in this document, it shows how IETF data models in
-{{?I-D.ietf-tvr-schedule-yang}} can fit into this framework and streamline the management and
-orchestration of network resources based on precise date and time parameters.
+This document provides the applicability statement on how the TVR(Time-Variant Routing) data model may
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:94]:
```suggestion
This document provides the applicability statement on how the TVR (Time-Variant Routing) data model may
```

**Review by QiufangMa — APPROVED** [review]

File: draft-wqb-tvr-applicability.md [hunk 4]
```diff
@@ -90,12 +91,13 @@ of resources to satisfy service demands or adhere to local networks policies.
 Many common building blocks are required for all these cases for adequate
 management of schedules and related scheduled actions.
 
-This document describes the applicability of various network management tools
-and data models that may be used for scheduling, to meet the requirements of a set
-of representative Time-Variant Routing (TVR) use cases described in {{?I-D.ietf-tvr-use-cases}}. By leveraging
-a reference framework presented in this document, it shows how IETF data models in
-{{?I-D.ietf-tvr-schedule-yang}} can fit into this framework and streamline the management and
-orchestration of network resources based on precise date and time parameters.
+This document provides the applicability statement on how the TVR (Time-Variant Routing) data model may
```
Comment (boucadair) [inline draft-wqb-tvr-applicability.md:94]:
```suggestion
This document provides an applicability statement on how the Time-Variant Routing (TVR) data model may
```

Comment (zhangli-abcd) [reply draft-wqb-tvr-applicability.md:94]: Great

**Pull request merged**